zetta_utils/mazepa/execution_state.py

```diff
@@ -9,6 +9,7 @@
 
 from zetta_utils import log
 
+from ..mazepa_addons.execution_tracker import read_execution_checkpoint
```

Comment:
OK, I only put it in addons because the logic was so similar to the `record_execution_run` method. That's also why I put the checkpoints under the same prefix as the execution info. Should I change that to something different, too?